### PR TITLE
feat: Add onSessionUpdate prop to get client instance

### PR DESCRIPTION
**Pull request by razor-x**
- Run npm update
- Update all dependencies
- feat: Add onInit to SeamProvider

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -94,6 +94,45 @@ export function App() {
 
 > Update the version in the script tag above with the exact version of this package you would like to use.
 
+#### Web component attributes and properties
+
+Each React component is defined as a custom element with a matching name in kebab-case.
```
Comment (xplato) [inline README.md:99]: The formatting and phrasing of this reads like a list—is it supposed to be a list?

**Review by xplato — APPROVED** [review]
Nice

**Comment by razor-x** [comment]
@xplato Any other ideas for the prop name? I stared with `onInit` and then moved to `onSessionUpdate`. I don't hate this name but would entertain some alternatives.

**Comment by xplato** [comment]
@razor-x I think both are good. Does the `client` only update once after init? You could do `onSessionInit` to make it clear that it should only change ~once (unless, of course, it's expected to update several times).

**Comment by razor-x** [comment]
> @razor-x I think both are good. Does the `client` only update once after init? You could do `onSessionInit` to make it clear that it should only change ~once (unless, of course, it's expected to update several times).

@xplato No, that's why I changed the name. The method will be called whenever the client session is refreshed or changed, say if the token expires, or the user identifier key prop changes (then a new client would be created)

**Pull request merged**